//
// Copyright 2017 Asylo authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package asylo;

import "asylo/enclave.proto";

message EnclaveTestString {
  optional string test_string = 1;
}

extend EnclaveConfig {
  optional EnclaveTestString enclave_config_test_string = 157658776;
}

extend EnclaveInput {
  optional EnclaveTestString enclave_input_test_string = 157658777;
}

extend EnclaveFinal {
  optional EnclaveTestString enclave_final_test_string = 157658778;
}

extend EnclaveOutput {
  optional EnclaveTestString enclave_output_test_string = 158466505;
}
